SPECS/kata-containers/kata-containers.spec

```diff
@@ -53,14 +53,10 @@ Source1:        https://github.com/%{name}/%{name}/releases/download/%{version}/
 Source2:        50-kata
 Source3:        mariner-build-uvm.sh
 Patch0:         0001-Merged-PR-9607-Allow-10-seconds-for-VM-creation-star.patch
-Patch1:         0002-Merged-PR-9671-Wait-for-a-possibly-slow-Guest.patch
-Patch2:         0003-Merged-PR-9805-Add-support-for-MSHV.patch
-Patch3:         0004-Merged-PR-9806-Fix-enable_debug-for-hypervisor.clh.patch
-Patch4:         0005-Merged-PR-9956-shim-avoid-memory-hotplug-timeout.patch
-Patch5:         runtime-reduce-uvm-high-mem-footprint.patch
-Patch6:         drop-mut-for-variables-that-are-not-mutated.patch
-Patch7:         0001-osbuilder-Add-support-for-CBL-Mariner.patch
-Patch8:         0001-Append-systemd-kernel-cmdline-params-for-initrd.patch
+Patch1:         0004-Merged-PR-9806-Fix-enable_debug-for-hypervisor.clh.patch
+Patch2:         0005-Merged-PR-9956-shim-avoid-memory-hotplug-timeout.patch
+Patch3:         runtime-reduce-uvm-high-mem-footprint.patch
+Patch4:         0001-Append-systemd-kernel-cmdline-params-for-initrd.patch
 
 BuildRequires:  golang
```

Comment:
Requires and Conficts sections:
- kata-agent, kata-ksm-throttler, kata-proxy, kata-runtime, kata-shim don't make sense to me. I don't think those packages exist. @danmihai1 do you remember?
- Requires: qemu-kvm-core: @danmihai1 should we leave this?

Reply:
qemu-kvm-core: The dependency known to me is qemu-virtiofsd. Potentially we can refine the dependency.